webdash-npm-scripts.html

```diff
@@ -301,7 +312,7 @@ <h2>Full output</h2>
             }
 
             stop(script) {
-                this.backend.post('stop', { script });
+                return this.backend.post('stop');
```

Comment:
Shouldn't we keep sending the `, {script}` here?
I was able to reproduce a bug with this change:
If you have 2 server scripts that are running, and you stop the first one, then the 2nd one
only the first one will be stopped, because you're not sending the `script`

Reply:
oh, this is typo 😬